android/daemon/src/main/java/org/mozilla/firefox/vpn/daemon/VPNService.kt

```diff
@@ -51,14 +51,15 @@ class VPNService : android.net.VpnService() {
         set(value: Int) {
             field = value
             if (value > -1) {
+                mConnectionTime = System.currentTimeMillis()
                 Log.i(tag, "Dispatch Daemon State -> connected")
                 mBinder.dispatchEvent(
                     VPNServiceBinder.EVENTS.connected,
                     JSONObject().apply {
+                        put("time", mConnectionTime)
                         put("city", mCityname)
                     }.toString()
                 )
-                mConnectionTime = System.currentTimeMillis()
```

Comment:
Why cannot we pass the connection time in the initialize method?

Reply:
That should have been fixed. 
We get the time from initialize only when the app closed/killed. Are you suggesting we should reinitialize the controller anytime we move from background/stopped -> started?